addons/project/models/project_task.py

```diff
@@ -1145,6 +1154,13 @@ def create(self, vals_list):
         # no track when the portal user create a task to avoid using during tracking
         # process since the portal does not have access to tracking models
         tasks = super(ProjectTask, self.with_context(mail_create_nosubscribe=True, mail_notrack=not self.env.su and self.env.user._is_portal())).create(vals_list)
+        if is_portal:
+            partner_id = self.env.user.partner_id.id
+            for task, child_ids in zip(tasks, child_ids_list):
+                if child_ids:
+                    task.sudo().message_subscribe([partner_id])
+                    task.write({'child_ids': child_ids})
```

Comment:
We have already message subscribed the parent task in L1199. If we add a subtask after, there will be no need to subscribe to it